compiler/crates/common/src/feature_flags.rs

```diff
@@ -108,6 +108,10 @@ pub struct FeatureFlags {
     /// Allow non-nullable return types from resolvers.
     #[serde(default)]
     pub allow_resolver_non_nullable_return_type: FeatureFlag,
+
+    /// Disable validation of the `edgeTypeName` argument on `@prependNode` and `@appendNode`.
+    #[serde(default)]
+    pub disable_edge_type_name_validation_on_declerative_connection_directives: bool,
```

Comment:
Could we use `FeatureFlag` here instead and then use [`is_enabled_for`](https://github.com/facebook/relay/blob/fdd270b61bb1101e4bd0056bfa83add0b420d257/compiler/crates/common/src/feature_flags.rs#L131) on the typename passed? That would allow us to implement a ratchet internally where we could improve things over time.

Reply:
Done :)